infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -458,20 +458,27 @@ private void handleAllocation(CorfuPayloadMsg<TokenRequest> msg,
         }
 
         // update the cache of conflict parameters
-        if (req.getTxnResolution() != null)
-            req.getTxnResolution().getWriteConflictParams().entrySet()
-                .stream()
-                    // for each entry
-                .forEach(txEntry ->
-                        // and for each conflict param
-                            txEntry.getValue().stream().forEach(conflictParam ->
-                                    // insert an entry with the new timestamp
-                                    // using the hash code based on the param
-                                    // and the stream id.
-                                    conflictToGlobalTailCache.put(
-                                            getConflictHashCode(txEntry
-                                                    .getKey(), conflictParam),
-                                            newTail - 1)));
+        if (req.getTxnResolution() != null) {
+            // Until we have per-stream conflict sets, any poisoned stream causes
+            // max wildcard to be updated
+            if (req.getTxnResolution().getPoisonedStreams().size() > 0) {
+                maxConflictWildcard = newTail - 1;
```

Comment:
that's not correct, otherwise the test wouldn't past (and the wildcard wouldn't work right?) On the contrary, it seems that the conflictWildcard overrides the tailMap and any transaction below the wildcard will be rejected?

from L219 in SequencerServer:
`                    if (v == null && maxConflictWildcard > txSnapshotTimestamp ) {
                        log.warn("ABORT[{}] conflict-key[{}](WILDCARD ts={})", txInfo, conflictParam,
                                maxConflictWildcard);
                        conflictKey.set(conflictParam);
                        response.set(TokenType.TX_ABORT_CONFLICT);
                    }`